trezorlib/tests/device_tests/test_msg_getaddress.py

```diff
@@ -103,6 +103,15 @@ def test_bch(self):
             == "bitcoincash:qzc5q87w069lzg7g3gzx0c8dz83mn7l02scej5aluw"
         )
 
+    @pytest.mark.xfail
+    @pytest.mark.capricoin
+    def test_cpc(self):
+        self.setup_mnemonic_allallall()
+        assert (
+            btc.get_address(self.client, "Capricoin", parse_path("m/44'/289'/0'/0/0"))
+            == "CMSgH7wq4kV9ogmSPB5rBmPceQJy3oA9Bu"
+        )
```

Comment:
are Capricoin changes affecting address generation? if not, we can omit this test